features/scripts/build_android_app.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 set -o errexit
 
+if [ -z "$FLUTTER_DIR" ]; then
+  FLUTTER_DIR="flutter"
+fi
+
```

Comment:
Could we echo the flutter location in use here, in case the variables don't make it through as we expect?